UIMediator/PanoramicDialogMediator.cpp

```diff
@@ -46,6 +46,17 @@ UIMediator::getPanSpectrumRange(qint64 &min, qint64 &max) const
   return false;
 }
 
+bool
+UIMediator::getPanSpectrumZoomRange(qint64 &min, qint64 &max, bool &noHop) const
+{
+  if (!this->m_ui->panoramicDialog->invalidRange()) {
+    this->m_ui->panoramicDialog->getZoomRange(min, max, noHop);
```

Comment:
Now that I see this, next time you see a class with private properties starting with `m_`, try not to use `this->` unless necessary. I used to abuse `this->` in the past (perhaps because too much influence from C) I eventually made the code for big classes unreadable. The solution was, since I still wanted to clearly see in a glimpse which variables were properties, which local and which global, to remove the `this->` (even for methods) and prefix private members (not methods) with `m_` (and globals with `g_`, although this is something I was already doing in C). You will see that, once in a while, when I have to work in an existing class written in old style, I leverage the changes to rewrite them in the new style.

So, no need for you to change that (I will do it at some point in the future), just wanted to let you know (and hopefully make your life easier)!